packages/profiling-node/scripts/check-build.js

```diff
@@ -0,0 +1,64 @@
+const fs = require('fs');
+const child_process = require('child_process');
+const binaries = require('./binaries.js');
+
+function clean(err) {
+  return err.toString().trim();
+}
+
+function recompileFromSource() {
+  // eslint-disable-next-line no-console
+  console.log('@sentry/profiling-node: Compiling from source...');
+  let spawn = child_process.spawnSync('npm', ['run', 'build:bindings:configure'], {
+    stdio: ['inherit', 'inherit', 'pipe'],
+    env: process.env,
+    shell: true,
+  });
+
+  if (spawn.status !== 0) {
+    // eslint-disable-next-line no-console
+    console.log('@sentry/profiling-node: Failed to configure gyp');
+    // eslint-disable-next-line no-console
+    console.log('@sentry/profiling-node:', clean(spawn.stderr));
+    return;
+  }
+
+  spawn = child_process.spawnSync('npm', ['run', 'build:bindings'], {
+    stdio: ['inherit', 'inherit', 'pipe'],
+    env: process.env,
+    shell: true,
+  });
+  if (spawn.status !== 0) {
+    // eslint-disable-next-line no-console
+    console.log('@sentry/profiling-node: Failed to build bindings');
+    // eslint-disable-next-line no-console
+    console.log('@sentry/profiling-node:', clean(spawn.stderr));
+    return;
+  }
+}
+
+if (fs.existsSync(binaries.target)) {
+  try {
+    // eslint-disable-next-line no-console
```

Comment:
It would be better to just disable no-console in the first line and in the whole file.

Reply:
Good call. I disabled it and added a comment in all 3 files :)